aliases/available/laravel.aliases.bash

```diff
@@ -3,7 +3,7 @@ about-alias 'laravel artisan abbreviations'
 
 # A list of useful laravel aliases
 
-alias laravel='${HOME?}/.composer/vendor/bin/laravel'
+alias laravel='${HOME?}/.config/composer/vendor/bin/laravel'
```

Comment:
Any chance some users still use the old location on older installations and will somehow lose functionality? I have a feeling this can benefit from an "if" and a fallback location if an older installation is found.

Reply:
@adbarbosa